app/assets/stylesheets/application.css.scss

```diff
@@ -90,7 +90,11 @@ a, a:visited, a:link {
 }
 
 .flash {
-    text-align: center;
+  text-align: center;
+  font-size: 1.3em;
+  -moz-box-shadow:    inset 0 0 10px #000000;
+ 	-webkit-box-shadow: inset 0 0 10px #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs